File: Facade/Golang/NonVegRestaurant.go
package main

type NonVegRestaurant struct {
	Menu *Menu
}

func NewNonVegRestaurant() *NonVegRestaurant {
	return &NonVegRestaurant{
		Menu: &Menu{
			Name: "Non Veg Menu",
		},
	}
}

func (nonVegRestaurant *NonVegRestaurant) getMenus() *Menu {
	return nonVegRestaurant.Menu
}

File: Facade/PHP/NonVegRestaurant.php
<?php

require_once "IRestaurant.php";
require_once "Menu.php";

class NonVegRestaurant implements IRestaurant {
    private Menu $menu;

    public function __construct()
    {
        $this->menu = new Menu("Non Veg Menu");
    }

    public function getMenus() {
        return $this->menu;
    }
}